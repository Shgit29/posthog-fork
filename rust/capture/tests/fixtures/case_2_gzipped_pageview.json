{
  "title": "gzipped pageview event",
  "path": "/e/?compression=gzip&ip=1&_=1694769302319&ver=1.78.5",
  "method": "POST",
  "content_encoding": "gzip",
  "content_type": "text/plain",
  "ip": "127.0.0.1",
  "now": "2023-09-15T09:15:02.322717+00:00",
  "historical_migration": false
}