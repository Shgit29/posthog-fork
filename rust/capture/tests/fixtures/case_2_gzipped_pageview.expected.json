[
  {
    "uuid": "018a981f-4b2f-791a-9eb1-67ae5b9ecf84",
    "distinct_id": "test-user-123",
    "ip": "127.0.0.1",
    "site_url": "http://localhost:8000",
    "data": "{\"uuid\": \"018a981f-4b2f-791a-9eb1-67ae5b9ecf84\", \"event\": \"$pageview\", \"properties\": {\"$current_url\": \"http://localhost:8000/\", \"$lib\": \"web\", \"$lib_version\": \"1.78.5\", \"distinct_id\": \"test-user-123\", \"token\": \"phc_test_token\"}, \"timestamp\": \"2023-09-15T09:15:02.319Z\"}",
    "now": "2023-09-15T09:15:02.322717+00:00",
    "sent_at": "2023-09-15T09:15:02.319000+00:00",
    "token": "phc_test_token"
  }
]